File: Beecrowd/8-Geometria_Computacional/2158-Ajudando_o_Tio_Cláudio-5/2158.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        long fp, fh;
        long edge, vertex;
        int i = 1;

        while (true) {
            try {
                String[] data = scanner.nextLine().split(" ");
                fp = Long.parseLong(data[0]);
                fh = Long.parseLong(data[1]);

                edge = (fp * 5 + fh * 6) / 2;
                vertex = 2 - (fp + fh) + edge;

                System.out.printf("Molecula #%d.:.%n", i);
                System.out.printf("Possui %d atomos e %d ligacoes%n%n", vertex, edge);
                i++;
            } catch (Exception e) {
                break;
            }
        }


    }
}

File: Beecrowd/8-Geometria_Computacional/2158-Ajudando_o_Tio_Cláudio-5/2158.cs
using System;

class solution{
    static void Main(){
        long fp, fh;
        long edge, vertex;
        int i =1;
        
        while(true){
            try{
                string[] data = Console.ReadLine().Split(' ');
                fp = long.Parse(data[0]);
                fh = long.Parse(data[1]);

                edge = (fp*5+fh*6)/2;
                vertex = 2-(fp+fh)+edge;
                Console.WriteLine($"Molecula #{i}.:.");
                Console.WriteLine($"Possui {vertex:0} atomos e {edge:0} ligacoes\n");
                i++;
            }
            catch{
                break;
            }

        }

        }
}